include/ignition/rendering/Scene.hh

```diff
@@ -954,6 +964,14 @@ namespace ignition
       public: virtual ParticleEmitterPtr CreateParticleEmitter(
                   unsigned int _id, const std::string &_name) = 0;
 
+      /// \brief Enable sky in the scene.
+      /// \param[in] _enabled True to enable sky
+      public: virtual void SetSkyEnabled(bool _enabled) = 0;
+
+      /// \brief Get whether the sky is enabled in the scene.
+      /// \return true to sky is enabled, false otherwise
+      public: virtual bool SkyEnabled() const = 0;
```

Comment:
The current sky API doesn't allow setting sky properties like cloud speed, humidity, ambient color... Are these supposed to be set through a custom background material?

Reply:
those are not supported yet so I held off on adding new APIs for setting these properties until we implement the features.

Reply:
Got it. The reason I asked is because it's not clear to me where those would go. Would they all be functions under `Scene`, like `Scene::SetSkyHumidity`, or do we need a `Sky` class to handle that, and the `Scene` only provides accessors to it (i.e. `Scene::CreateSky` / `Scene::Sky`)?

Reply:
Since there should only be one global sky, I am leaning towards something like `Scene::SetSkyProperties(const SkyConfiguration &_config)` function. Open to suggestions though

Reply:
Got it, that sounds good to me and I think it fits well with the functions added in this PR :+1: